mmv1/third_party/terraform/services/dataflow/resource_dataflow_flex_template_job_test.go.erb

```diff
@@ -439,6 +439,43 @@ func TestAccDataflowFlexTemplateJob_withProviderDefaultLabels(t *testing.T) {
 	})
 }
 
+func TestAccDataflowFlexTemplateJob_migration(t *testing.T) {
```

Comment:
What makes the tests interesting- what new behaviour are they covering?

Reply:
They don't cover new behaviour, just to verify if the upgrade works or not.

Reply:
Removed the tests.